cmd/kubelet/app/options/options.go

```diff
@@ -210,7 +210,6 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.BabysitDaemons, "babysit-daemons", s.BabysitDaemons, "If true, the node has babysitter process monitoring docker and kubelet.")
 	fs.MarkDeprecated("babysit-daemons", "Will be removed in a future version.")
 	fs.Int32Var(&s.MaxPods, "max-pods", s.MaxPods, "Number of Pods that can run on this Kubelet.")
-	fs.Int32Var(&s.NvidiaGPUs, "experimental-nvidia-gpus", s.NvidiaGPUs, "Number of NVIDIA GPU devices on this node. Only 0 (default) and 1 are currently supported.")
```

Comment:
HI @vishh , This PR is actually needed to fix the original PR that had been merged already which only support 1 GPU only. So, think of it as bug fix rather than feature. I am working at eBay to support K8S for ML with GPUs so would love to see this in for v1.6 too